appengine/accessible/generators-javascript-music.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * @license
+ * Visual Blocks Language
+ *
+ * Copyright 2016 Google Inc.
+ * https://github.com/google/blockly-games
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Generating JavaScript for music blocks.
+ * @author sll@google.com (Sean Lip)
+ */
+'use strict';
+
+goog.provide('Blockly.JavaScript.music');
+
+goog.require('Blockly.JavaScript');
+
+Blockly.JavaScript['music_play_random_note'] = function(block) {
+  // Play a single note.
```

Comment:
A better comment might be:

// Play a single random note.
// TODO(madeeha): Make this more random.

(It's OK to leave the TODO there. I'll take care of it in a future pass.)